Extensions/show_more.js

```diff
@@ -660,48 +651,6 @@ XKit.extensions.show_more = new Object({
 
 		}
 
-		if (XKit.extensions.show_more.preferences.enable_anon.value) {
```

Comment:
This is now a feature in vanilla tumblr, right?

Reply:
yep, i think this is a long leftover option for a feature that xkit doesn't even have anymore

Reply:
yeah, even with xkit's classic user menu the vanilla askbox gets pulled up, which has an anonymous toggle